jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -402,38 +402,18 @@ def test_add_mul(self, harness: primitive_harness.Harness):
 
   @primitive_harness.parameterized(primitive_harness.lax_min_max)
   def test_min_max(self, harness: primitive_harness.Harness):
-    expect_tf_exceptions = False
-    dtype = harness.params["dtype"]
```

Comment:
In the design doc you made the strong point that this code occurs 3 times. I am curious if the technique works to remove all those uses? I suspect that it will work in some sense, but the error message is not going to be about `reduce_window` but about `max`.

Reply:
Correct, this is my interpretation as well. Which is fine, since the limitation is really coming from `max`.